app/helpers/users_helper.rb

```diff
@@ -203,4 +204,10 @@ def strong_password_definition
   def show_provider_sso_status_for_user?
     current_account.provider_can_use?(:provider_sso) && current_account.self_authentication_providers.published.any?
   end
+
+  def permission_groups_summary(user = current_user)
```

Comment:
This could be in https://github.com/3scale/porta/blob/master/app/decorators/user_decorator.rb alternatively. I preferred having it here in the end because I don't see us including this string in the API response anyway.